contrib/slurm/slurmexecutor.py

```diff
@@ -0,0 +1,297 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+# SPDX-FileCopyrightText: 2024 Levente Bajczi
+# SPDX-FileCopyrightText: Critical Systems Research Group
+# SPDX-FileCopyrightText: Budapest University of Technology and Economics <https://www.ftsrg.mit.bme.hu>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import queue
+import re
+import subprocess
+import sys
+import tempfile
+import threading
+import time
+
+from benchexec import benchexec, BenchExecException, tooladapter
+from benchexec.util import ProcessExitCode
+
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
+
+WORKER_THREADS = []
+STOPPED_BY_INTERRUPT = False
+
+
+def init(config, benchmark):
+    tool_locator = tooladapter.create_tool_locator(config)
+    benchmark.executable = benchmark.tool.executable(tool_locator)
+    benchmark.tool_version = benchmark.tool.version(benchmark.executable)
+
+    logging.info("Using %s version %s.", benchmark.tool_name, benchmark.tool_version)
+
+
+def get_system_info():
+    return None
+
+
+def execute_benchmark(benchmark, output_handler):
+    num_of_cores = benchmark.rlimits.cpu_cores
+    mem_limit = benchmark.rlimits.memory
+    run_sets_executed = 0
```

Comment:
This seems unused.

Reply:
I reworked argument passing a bit because querying resource limits this early on was not a good idea, as we pass `benchmark` along to the worker, and we can query it there instead.
For the `run_sets_executed`: it was indeed not used, I removed it. However, I noticed that it also present in `localexecution.py`, where it is not used, either - maybe we should remove it there, as well?